src/accounts/AccountPage.tsx

```diff
@@ -100,9 +114,35 @@ const AccountAboutPage: FC = () => {
             text="Save"
           />
         </FlexBox>
+        <hr style={dividerStyle} />
+        <h4
+          data-testid="account-settings--header"
+          className="account-settings--header"
+        >
+          Deactivate Account
+        </h4>
+        <div style={labelStyle}>
+          If you decide to deactivate this account, all your writes, queries,
+          and tasks will be suspended immediately.
+        </div>
+        <FlexBox direction={FlexDirection.Row} margin={ComponentSize.Large}>
+          <Button
+            onClick={() => setDeactivateAccountVisible(true)}
```

Comment:
anonymous functions like this will always cause re-renders. this is a small page so it's probably not a huge deal right now, but something to be aware of, especially in class components

Reply:
Ahh!! I had that on my list to fix before I requested a review but I totally forgot, I can fix this.